Automattic/wp-calypso
# Move sites/SITE/users data-layer to reflect route

Moves a data-layer middleware to reflect its API route according [to convention](https://github.com/Automattic/wp-calypso/tree/master/client/state/data-layer#file-structure):

> The file which handles requests to the `/sites/[ siteID ]/posts` endpoint should live at `state/data-layer/wpcom/sites/posts/index.js`

## Testing
1. Tests pass
1. Ensure middleware continues to work correctly.
   * Used via query component `QueryUsers`, which only uses this via post revisions. Try editing a post written by another person, clicking "History" at the top of the editor, and you should see the requests and the action fired (`actionLog.filter(/USER/)`)
   * Try manual dispatch: `dispatch( { type:'USERS_REQUEST', siteId: state.ui.selectedSiteId } )`
